### PR TITLE
Dashboard Admin - Logs

### PR DESCRIPTION
**Pull request by jpennors**
Add Time to Logs table in Dashboard admin
Refactor files with linter

### CODE REVIEW

File: react/app-admin/components/OverviewPage/OverviewPage.tsx [hunk 1]
```diff
@@ -1,25 +1,21 @@
 import * as React from 'react';
 import { Message, Loader, Table, Label } from 'semantic-ui-react';
 import { ActionMessage } from 'tmw-admin/components/ActionMessage';
-import { VisitorsChart } from 'tmw-admin/components/OverviewPage'
+import { VisitorsChart } from 'tmw-admin/components/OverviewPage';
 import { PageHeader } from 'tmw-admin/components/PageHeader';
 import { serializeLogsFromAPI, serializeVisitorStatsFromAPI } from 'tmw-admin/utils/api-serialize';
 import { ajaxGet, ajaxPost } from 'tmw-common/utils/ajax';
-import { getApiDateFormat } from '../../../common/utils/date';
+import { getApiDateFormat, getTimeFromApiDate } from '../../../common/utils/date';
```
Comment (lancelotimb) [inline react/app-admin/components/OverviewPage/OverviewPage.tsx]: I would put the absolute path here to be consistent with the rest of the app (+ it's easier if we move the file later)

File: react/common/utils/date.ts [hunk 2]
```diff
@@ -1,9 +1,14 @@
-
 export function getApiDateFormat(date: Date) {
-
-    const day = ("0" + (date.getDate())).slice(-2);
-    const month_number = ("0" + (date.getMonth() + 1)).slice(-2)
+    const day = ('0' + date.getDate()).slice(-2);
+    const month_number = ('0' + (date.getMonth() + 1)).slice(-2);
```
Comment (lancelotimb) [inline react/common/utils/date.ts]: Not a big deal but we should ensure every variable is in `camelCase` in the Javascript codebase, could you rename this one `monthNumber`?

**Review by lancelotimb — APPROVED** [review]

File: react/app-admin/components/OverviewPage/OverviewPage.tsx (hunk 1 above)
Comment (jpennors) [reply react/app-admin/components/OverviewPage/OverviewPage.tsx]: You're right, I just did it ! 👍

File: react/common/utils/date.ts (hunk 2 above)
Comment (jpennors) [reply react/common/utils/date.ts]: Did it too 👍

**Pull request merged**